### PR TITLE
Added verbs and subjects

### PR DESCRIPTION
**Pull request by codingneko**
Added verbs and subjects

### CODE REVIEW

File: arrays.ts [hunk 1]
```diff
@@ -60,7 +60,17 @@ export const VERBS: Verb[] = [
...
   'assembled',
   'bapped',
-  "Blåhaj'd"
+  "Blåhaj'd",
+  { word: 'fucked', cw: 'lewd, aggressive' },
+  'added',
+  'subtracted',
+  'modded',
+  'boymodded',
+  'girlmodded',
```
Comment (theresnotime) [inline arrays.ts]: awawawa did you mean `boymoded` and `girlmoded`? :3 

```suggestion
  'boymoded',
  'girlmoded',
```

Comment (codingneko) [reply arrays.ts]: Oh just one D? I've always spelled it with 2 ds xd but sure, will correct!

File: arrays.ts [hunk 2]
```diff
@@ -62,6 +62,16 @@ export const VERBS: Verb[] = [
...
   "Blåhaj'd",
+  { word: 'fucked', cw: 'lewd, aggressive' },
+  'added',
+  'subtracted',
+  'modded',
+  'boymoded',
+  'girlmoded',
+  'gender swapped',
+  'rooted',
+  'rotated',
```
Comment (olivvybee) [inline arrays.ts]: It seems I accidentally added `rotated` separately and caused the duplicate detection to get upset, sorry about that 😔

```suggestion
```

Comment (codingneko) [reply arrays.ts]: No worries, fixed :3

**Pull request merged**